lib/mix/lib/mix/compilers/elixir.ex

```diff
@@ -158,12 +173,25 @@ defmodule Mix.Compilers.Elixir do
 
     kind   = detect_kind(module)
     source = Path.relative_to(source, cwd)
-    files  = get_external_resources(module, cwd)
+    source_files  = get_external_resources(module, cwd)
+
+    Agent.cast pid, fn {modules, files, sources} ->
+      new_module =
+        module(
+          module: module,
+          kind: kind,
+          source: source,
+          beam: beam,
+          binary: binary
+        )
```

Comment:
Wdyt about putting the `module(` part on the same line as `new_module`?

```elixir
new_module = module(
  module: module,
  ...
)
```

It looks pretty common to me this way. Wdyt? I'd ping @lexmag on this as well as I'm not 100% sure, so a second opinion would be good :)